src/diffusers/pipelines/pag/pag_utils.py

```diff
@@ -122,7 +123,7 @@ def _apply_perturbed_attention_guidance(self, noise_pred, do_classifier_free_gui
         else:
             noise_pred_text, noise_pred_perturb = noise_pred.chunk(2)
             noise_pred = noise_pred_text + pag_scale * (noise_pred_text - noise_pred_perturb)
-        return noise_pred
+        return noise_pred, noise_pred_text
```

Comment:
seems like we broke the tests; I looked up the import of PAGMixin on github, not many but some so let's try to not making a breaking change here

we can add a argument `return_pred_text`; default to `False` but set `True` when calling from our pipelines.